src/mtserver.pri

```diff
@@ -8,7 +8,7 @@ CONFIG += c++11
 isEmpty(TPOOL_PATH) {
     error("NrThreadPool path not defined! Please define the TPOOL_PATH variable with the location of nrThreadPool.pri before including this file (mtserver.pri)")
 }
-include($$TPOOL_PATH/nrThreadPool.pri)
+include($$TPOOL_PATH/src/nrThreadPool.pri)
```

Comment:
I do not remember which version of tpool adds the `src` folder.
Adding `src/` to the hardcoded path breaks builds using older version.
Actually you can make it working by set `TPOOL_PATH` to `$$TPOOL_PATH/src`, but I agree we should move forward.

@alienpenguin opinions?